TicTacToe.Foundation/Figures/FigureFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using TicTacToe.Foundation.Interfaces;
+
+namespace TicTacToe.Foundation.Figures
+{
+    public class FigureFactory : IFigureFactory
+    {
+        public IFigure CreateFigure(FigureType figureType)
+        {
+            switch (figureType)
+            {
+                case FigureType.Circle:
+                    return new Circle();
+                case FigureType.Cross:
+                    return new Cross();
+                default:
+                    throw new ArgumentException("Invalid figureType");
```

Comment:
Please, throw out of range exception.
Add figure type value param to the message.